components/tools/OmeroPy/src/omero/util/script_utils.py

```diff
@@ -30,8 +30,11 @@
 from numpy import add, array, asarray, fromstring, reshape, zeros
 from os.path import exists
 
+
```

Comment:
:+1: I think there was at least another double-blank introduced. Apart from that this looks good, we'll see when CI is green.